server.js

```diff
@@ -23,15 +23,106 @@ mongoose.Promise = Promise
 const port = process.env.PORT || 8080
 const app = express()
 
+
+
+const Book = mongoose.model('Book', {
+  bookId: {
+    type: Number
+  },
+  title: {
+    type: String
+  },
+  authors: {
+    type: String
+  },
+  average_rating: {
+    type: Number
+  },
+  isbn: {
+    type: String,
+    uniqe: true
+  },
+  isbn13: {
+    type: String
+  },
+  language_code: {
+    type: String
+  },
+  ratings_count: {
+    type: Number
+  },
+  ext_reviews_count: {
+    type: Number
+  }
+})
+
+if (process.env.RESET_DB) {
+  const addBooksToDataBase = async () => {
+    await Book.deleteMany({})
+
+    booksData.forEach((book) => {
+      new Book(book).save()
+    })
+  }
+
+  addBooksToDataBase()
+}
+
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
+
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Endpoints: /books (search by title, author or use sort_by=average_rating to get the books sorted), /books/:isbn (search for a single book)')
 })
 
+app.get('/books', (req, res) => {
+  const { title, author, sort_by } = req.query
+
+  const sort = {}
+
+  // Check if sort by is a descending sortable
+  if (sort_by && ['average_rating'].includes(sort_by)) {
```

Comment:
Why `['average_rating'].includes(sort_by)` instead of `'average_rating' === sort_by`?